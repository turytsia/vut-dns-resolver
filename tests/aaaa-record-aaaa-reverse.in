-r -t -x -s 2001:4860:4860::8888 2a00:1450:400d:806::2004