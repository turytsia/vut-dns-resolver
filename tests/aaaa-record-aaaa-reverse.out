Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
 4.0.0.2.0.0.0.0.0.0.0.0.0.0.0.0.6.0.8.0.d.0.0.4.0.5.4.1.0.0.a.2.ip6.arpa., PTR, IN
Answer section (2)
 4.0.0.2.0.0.0.0.0.0.0.0.0.0.0.0.6.0.8.0.d.0.0.4.0.5.4.1.0.0.a.2.ip6.arpa., PTR, IN, 0, bud02s33-in-x04.1e100.net.
 4.0.0.2.0.0.0.0.0.0.0.0.0.0.0.0.6.0.8.0.d.0.0.4.0.5.4.1.0.0.a.2.ip6.arpa., PTR, IN, 0, bud02s21-in-x04.1e100.net.
Authority section (0)
Additional section (0)
